<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="1">
     <widget class="QSlider" name="horizontalSliderRotate">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimum">
       <number>-18</number>
      </property>
      <property name="maximum">
       <number>18</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="tracking">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QSlider" name="verticalSliderRotate">
      <property name="minimum">
       <number>-18</number>
      </property>
      <property name="maximum">
       <number>18</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="GLWidget" name="widgetScene" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="groupBoxView">
         <property name="title">
          <string>View</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="radioButtonEditor">
            <property name="text">
             <string>Editor</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonCamera">
            <property name="text">
             <string>Camera</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelPickedObject">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>160</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="text">
          <string>No object selected</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="indent">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="6" column="0" alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="pushButtonDefault">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Default</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QGroupBox" name="groupBoxProjection">
         <property name="title">
          <string>Projection</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="radioButtonPerspective">
            <property name="text">
             <string>Perspective</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonOrthographic">
            <property name="text">
             <string>Orthographic</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QScrollBar" name="verticalScrollBarZoom">
      <property name="minimum">
       <number>-2000</number>
      </property>
      <property name="maximum">
       <number>2000</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>880</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionScreenshot"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuPickedObject">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Picked Object</string>
     </property>
     <addaction name="actionTranslatePickedObject"/>
     <addaction name="actionScalePickedObject"/>
     <addaction name="actionRotatePickedObject"/>
     <addaction name="actionDeletePickedObject"/>
    </widget>
    <widget class="QMenu" name="menuCamera">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Camera</string>
     </property>
     <addaction name="actionEditPositionCamera"/>
    </widget>
    <addaction name="actionLoadObject"/>
    <addaction name="actionLoadBackground"/>
    <addaction name="menuPickedObject"/>
    <addaction name="menuCamera"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAboutHelp"/>
    <addaction name="actionContentsHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAboutHelp">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpenDefaultScene">
   <property name="text">
    <string>Open Default Scene</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionScreenshot">
   <property name="text">
    <string>Screenshot</string>
   </property>
  </action>
  <action name="actionEditor_View">
   <property name="text">
    <string>Editor View</string>
   </property>
  </action>
  <action name="actionEditorView">
   <property name="text">
    <string>Editor</string>
   </property>
  </action>
  <action name="actionCameraView">
   <property name="text">
    <string>Camera</string>
   </property>
  </action>
  <action name="actionPerspectiveProjection">
   <property name="text">
    <string>Perspective</string>
   </property>
  </action>
  <action name="actionObliqueProjection">
   <property name="text">
    <string>Oblique</string>
   </property>
  </action>
  <action name="actionLoadObject">
   <property name="text">
    <string>Load Object</string>
   </property>
  </action>
  <action name="actionLoadBackground">
   <property name="text">
    <string>Load Background</string>
   </property>
  </action>
  <action name="actionObjects">
   <property name="text">
    <string>Objects</string>
   </property>
  </action>
  <action name="actionTranslatePickedObject">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Translate</string>
   </property>
  </action>
  <action name="actionScalePickedObject">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Scale</string>
   </property>
  </action>
  <action name="actionRotatePickedObject">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rotate</string>
   </property>
  </action>
  <action name="actionNewLight">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionDeleteLight">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionModifyLight">
   <property name="text">
    <string>Modify Light Position</string>
   </property>
  </action>
  <action name="actionEditPositionCamera">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Edit Position</string>
   </property>
  </action>
  <action name="actionContentsHelp">
   <property name="text">
    <string>Contents</string>
   </property>
  </action>
  <action name="actionExit_Full_Screen">
   <property name="text">
    <string>Exit Full Screen</string>
   </property>
  </action>
  <action name="actionShowFullScreen">
   <property name="text">
    <string>Show</string>
   </property>
  </action>
  <action name="actionExitFullScreen">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionDeletePickedObject">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header location="global">GLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
